defaults/main.yml

```diff
@@ -1,7 +1,32 @@
+# note: the booleans are quoted otherwise they are rendered capitalised.
 ---
+buildkite_agent_major_version: 3
 buildkite_agent_conf_dir: "/etc/buildkite-agent"
 buildkite_agent_token: ""
-buildkite_agent_name: "buildkite-agent"
+buildkite_agent_debug: "false"
+buildkite_agent_enabled: true
+# if you're building a golden image and don't want it to connect to
+# buildkite during build ptocess, set this to 'stopped'.
+buildkite_agent_state: "started"
+
+# v2 https://buildkite.com/docs/agent/v2/configuration
+buildkite_agent_name: "buildkite-agent" # v3 default: '%hostname-%n'
 buildkite_agent_meta_data: "queue=default"
 buildkite_agent_meta_data_ec2_tags: "true"
-buildkite_agent_debug: "false"
+
+# v3 https://buildkite.com/docs/agent/v3/configuration
+buildkite_agent_priority: 1
+buildkite_agent_queue: "default"
+buildkite_agent_tags:
+  - "queue={{ buildkite_agent_queue }}"
+buildkite_agent_tags_from_ec2: "false"
+buildkite_agent_tags_from_ec2_tags: "false"
+buildkite_agent_tags_from_gcp: "false"
+buildkite_agent_bootstrap_script: ""
```

Comment:
Should this default to `buildkite-agent bootstrap` based on the comment in https://github.com/azavea/ansible-buildkite-agent/pull/3/files#diff-c065fbb7e1b1961b255807b0096329f5R22?

Reply:
Yes, now that I read that doc. Thanks.